package.json

```diff
@@ -17,22 +17,22 @@
   "license": "MIT",
   "dependencies": {
     "alphaify": "^1.0.0",
-    "bowser": "^0.7.3",
-    "canvas-composite-types": "^1.0.0",
-    "d3": "^3.5.5",
+    "bowser": "^1.0.0",
+    "canvas-composite-types": "^1.0.4",
```

Comment:
I don't think there is a `canvas-composite-types` with version `1.0.4`. This is causing `npm install` to fail:

````lang=sh
npm ERR! Darwin 14.5.0
npm ERR! argv "node" "/Users/victorpowell/.nvm/v0.10.32/bin/npm" "install"
npm ERR! node v0.10.32
npm ERR! npm  v3.7.2

npm ERR! No compatible version found: canvas-composite-types@^1.0.4
npm ERR! Valid install targets:
npm ERR! 1.0.0
````